scripts/ci/cmake/ci-suse-pgi.cmake

```diff
@@ -8,9 +8,9 @@ env_module(load pgi)
 set(ENV{CC}  pgcc)
 set(ENV{CXX} pgc++)
 set(ENV{FC}  pgfortran)
-#set(ENV{CFLAGS} -Werror)
+set(ENV{CFLAGS} -Werror)
 set(ENV{CXXFLAGS} --brief_diagnostics)
```

Comment:
@chuckatkins the PGI C compiler does not understand `-Werror`.  Do you know the correct options to use here for C and C++?

Reply:
AFAIK it doesn't have such an option.